uint16[13] data
